commerce_coordinator/apps/commercetools/views.py

```diff
@@ -48,13 +53,15 @@ def post(self, request):
 
         default_params = {
             'email_opt_in': True,  # ?? Where?
-            'order_number': order.order_number or order.id,
```

Comment:
im ok with this on paper as order id... but once we set order number this may need to change.

Reply:
We are setting order number but that is in the format `cart-id-{cartId}`. We decided to change this to order id since this information is sent to `lms` for fulfillment and also this order number is shown in the support tools AFAIK so it makes more sense to send unique identifier of order in this field.